packages/terra-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -178,6 +178,22 @@ function WorklistDataGrid(props) {
   const gridReceivedFocus = useRef(false);
   const gridHasFocus = document.getElementById(`${id}-worklist-data-grid-container`)?.contains(document.activeElement);
 
+  const makeWorklistDataGridColumns = (column) => ({
```

Comment:
This function should be capable of iterating over the map as well. Then all you have to do is pass the necessary column array.